include/picongpu/fields/incidentField/profiles/GaussianPulse.hpp

```diff
@@ -387,6 +387,22 @@ namespace picongpu::fields::incidentField
             }
         };
 
+        template<typename T_Func>
+        struct FreeEnvelope
+        {
+            static constexpr float_X TIME_SHIFT = 0.0_X;
```

Comment:
What is this doing here? And is it shadowing the `float_64 TIME_DELAY` from the `BaseParam`?